src/theme.ts

```diff
@@ -91,15 +96,31 @@ export const BREAKPOINTS = {
   mobile: 600,
   tablet: 999,
   desktop: 1000,
+  large: 1600,
 };
 
-const desktopQuery = `(min-width: ${BREAKPOINTS.desktop}px)`;
 export const breakpoints = {
-  desktop: (style: FlattenSimpleInterpolation) => css`
-    @media screen and ${desktopQuery} {
-      ${style}
-    }
-  `,
+  mobile(...args: Parameters<typeof css>) {
+    return css`@media(max-width: ${BREAKPOINTS.mobile}px) { ${css(...args)} }`;
+  },
+  tablet(...args: Parameters<typeof css>) {
+    return css`@media(max-width: ${BREAKPOINTS.tablet}px) { ${css(...args)} }`;
+  },
+  desktop(...args: Parameters<typeof css>) {
+    return css`@media(min-width: ${BREAKPOINTS.desktop}px) { ${css(...args)} }`;
+  },
+  only: {
+    mobile(...args: Parameters<typeof css>) {
+      return css`@media(max-width: ${BREAKPOINTS.mobile}px) { ${css(...args)} }`;
+    },
+    tablet(...args: Parameters<typeof css>) {
+      return css`@media(min-width: ${BREAKPOINTS.mobile + 1}px) and (max-width: ${BREAKPOINTS.tablet}px) { ${css(...args)} }`;
+    },
+  },
+  margins: {
+    mobile: '8px',
+    tablet: '24px',
+  },
```

Comment:
Sorry for the constant theme changes, I realized in this branch we probably just wanted to copy the actual Tutor mixins, so this gets all the ones we're using in styles so far.